### ISSUE TITLE
AudioKit doesn't sound on OS X project and OSX/iOS playground

### ISSUE THREAD

**Issue by mtfrctl** [audiokit/AudioKit#336]
AudioKit example app for OSX doesn't sound in my environment (bottom of this article).
I directly downloaded audiokit/AudioKit repo from github, built frameworks by build_frameworks.sh, and tried to run AudioKit-master > Examples > OSX > HelloWorld > HelloWorld.xcodeproj.

Soon a window below appeared and then I clicked the button "Play Sine Wave" but
1. AKOutputWaveformPlot view wasn't rendered
2. No sound was occur
<img width="480" alt="2016-04-09 21 37 17" src="https://cloud.githubusercontent.com/assets/988216/14403686/5594f660-fe9c-11e5-8f13-0dcbdbfd7786.png">

No warning/error on editor/console was displayed.

Another way of installations I tried below are buildable and runnable but didn't work fine too.
1. Create a new empty OSX Cocoa project
2.a. Install AudioKit using CocoaPods
or
2.b. Install AudioKit by throwing AudioKit.framework into Embedded Binaries
3. Write exactly same code of HelloWorld (below)
4. Make a button and a custom view for AKOutputWaveformPlot in Interface Builder
5. Apply "AKOutputWaveformPlot" to the field of Identity inspector > Custom Class > Class
6. Apply "AudioKit" to the field of Identity inspector > Custom Class > Module
7. Connect the custom view to the corresponding property
8. Run
```
import Cocoa
import AudioKit

class ViewController: NSViewController {

    var oscillator = AKOscillator()

    @IBOutlet var plot: AKOutputWaveformPlot!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        AudioKit.output = oscillator
        AudioKit.start()
    }
    
    @IBAction func toggleSound(sender: NSButton) {
        if oscillator.isPlaying {
            oscillator.stop()
            sender.title = "Play Sine Wave"
        } else {
            oscillator.amplitude = random(0.5, 1)
            oscillator.frequency = random(220, 880)
            oscillator.start()
            sender.title = "Stop Sine Wave at \(Int(oscillator.frequency))Hz"
        }
        sender.setNeedsDisplay()
    }
}
```

Playground project for OSX and iOS are also unworkable.

**And the interesting point is that the HelloWorld samples for the iOS and the tvOS worked (sounds and rendered) fine on simulator.**

Can these problems are reproduced in your environment?

I believe it might be not the problem of OSX's Sound environment settings or Audio MIDI setup, but I paste screenshots just in case.
<img width="668" alt="2016-04-09 22 00 54" src="https://cloud.githubusercontent.com/assets/988216/14403770/c9366944-fe9e-11e5-8872-f8ee27a46081.png">
<img width="678" alt="2016-04-09 22 01 56" src="https://cloud.githubusercontent.com/assets/988216/14403771/cf997dd0-fe9e-11e5-8b56-c99a5e834f37.png">

About my Mac book pro retina 15 inch is below.
Hardware Overview:
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro11,3
  Processor Name:	Intel Core i7
  Processor Speed:	2.6 GHz
  Number of Processors:	1
  Total Number of Cores:	4
  L2 Cache (per Core):	256 KB
  L3 Cache:	6 MB
  Memory:	16 GB
  Boot ROM Version:	MBP112.0138.B17
  SMC Version (system):	2.19f12

Please forgive me if there is some kind of mistake.
I am looking forward to your reply.
Regards.

**Comment by aure** [comment]
First of all, thanks for the very thorough bug report.  Secondly, I want to confirm that I am getting the same problem as you.  The tests passed because the project builds, but no sound is made.  I'm working on this right now.

**Comment by aure** [comment]
This is fixed in the develop branch.  If you don't want to switch branches, just copy the AudioKit.swift file from here:  https://github.com/audiokit/AudioKit/blob/develop/AudioKit/Common/Internals/AudioKit.swift
over yours.  We will make a minor revision release for this bug since it is killing all OSX projects.

**Comment by aure** [comment]
And once again.... THANK YOU!!!!!!!!

**Comment by aure** [comment]
I also notice in your report that you couldn't get the iOS or OSX playgrounds to work.  Make sure you're running them like this: https://vimeo.com/157204402  and if you're still having problems, open up a new issue or we can also do a screenhero session on your computer to get them working.

**Comment by mtfrctl** [comment]
I appreciate such a quick response!
Soon I downloaded the develop branch, run Example, and it worked well!!

The problem seems to be occurred because of starting an audio engine in a debug scope of the library(AudioKit.swift), right? I guess it might be hardly noticeable...

The problem of playground programs were also solved after following procedure below.
1. If there is another window of Xcode, close it, or simply quit Xcode
2. Open "AudioKit For iOS.xcodeproj" of the new develop branch (restarting Xcode and using new code)
3. Build and Run
Before doing 1, the program didn't work, but now it works fine! :-)

Again, thank you so much for your fast fixing.
I'm rooting for you and your great work!!
